source/ui/SXRMB/SXRMBScanConfigurationView.cpp

```diff
@@ -30,7 +30,7 @@ void SXRMBScanConfigurationView::updateFluorescenceDetectorComboBox(SXRMB::Fluor
 QComboBox *SXRMBScanConfigurationView::createFluorescenceComboBox()
 {
 	QComboBox *newComboBox = new QComboBox;
-	newComboBox->insertItem(0, "None");
+	newComboBox->insertItem(0, "Transmission");
```

Comment:
Yes, I remember you saying that.  In this particular case I think the changes you implemented are the better way to go.  If need be, explain to Quanfeng why we did it this way.  Emphasize that they are always doing transmission and this combo box is meant to add an XRF detector as well.